warehouse/templates/pages/help.html

```diff
@@ -328,6 +330,11 @@ <h3 id="contributing">{{ contributing() }}</h3>
           <em>Stay updated</em>: You can also follow the ongoing development of the project on the <a href="https://mail.python.org/mailman/listinfo/distutils-sig">Distutils-sig mailing list</a> and the <a href="https://groups.google.com/forum/#!forum/pypa-dev">PyPA Dev message group</a>.
         </p>
         {{ code_of_conduct() }}
+
+        <h3 id="upcoming-changes">{{ upcoming_changes() }}</h3>
+        <p>
+          Subscribe to <a href="https://mail.python.org/mm3/mailman3/lists/pypi-announce.python.org/">the pypi-announce mailing list</a> and <a href="https://pyfound.blogspot.com/search/label/pypi">read PyPI-related blog posts on the PSF blog</a> (<a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi">Atom feed</a>, <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi?alt=rss">RSS feed</a>).
```

Comment:
I recommend we word this a bit differently:

```html
Changes to PyPI are generally announced on both the the <a href="https://mail.python.org/mm3/mailman3/lists/pypi-announce.python.org/">pypi-announce mailing list</a> and the <a href="https://pyfound.blogspot.com/search/label/pypi">PSF blog</a> under the label "pypi". The PSF blog also has <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi">Atom</a> and <a href="https://pyfound.blogspot.com/feeds/posts/default/-/pypi?alt=rss">RSS</a> feeds for the "pypi" label.
```

Also please make sure all external links have `rel="noopener"` and `target="_blank"`.

Reply:
Agreed.